CMSgov/cms-carts-seds
# MDCT-79 - Local Auth over CARTS UI

# Description

https://qmacbis.atlassian.net/browse/MDCT-79

Set up local auth for the main branch, a la QMR on top of the CARTS UI

## How to test

Run locally, users can be found in the users.json file. The UI makes an authenticated hello world call to the backend on successful login using the new pattern of auth. Before logging in, on the LocalLogin page, the same call is made and should 403.

You will need updated .env files to point at the CARTS environment.

## Dependencies
None

# Get it done

Still need to get the env configured to point at the CARTS Dev stack, but that can be done in env files, and updating the default users.

## Code authors checklist


## Reviewers Checklist (Two different people)


## Assignee
